src/EditorFeatures/Core/InlineRename/InlineRenameUIOptions.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.CodeAnalysis.Editor.InlineRename
 {
     internal sealed class InlineRenameUIOptions
```

Comment:
>InlineRenameUIOptions [](http://example.com/codeflow?start=26&length=21)

Please, rename to `InlineRenameUIOptionsStorage`

Reply:
I will rename all the similar classes